formats/src/main/scala/overflowdb/formats/ExporterMain.scala

```diff
@@ -37,6 +38,7 @@ object ExporterMain {
           val exporter: Exporter = format match {
             case Format.Neo4jCsv => Neo4jCsvExporter
             case Format.GraphMl => GraphMLExporter
+            case Format.GraphSon => GraphSONExporter
```

Comment:
```suggestion
            case Format.GraphML => GraphMLExporter
            case Format.GraphSON => GraphSONExporter
```

I'm having second thoughts on the naming convention I started here... thoughts?

Reply:
Yes I think this will be consistent with the class names